tensorrt_llm/llmapi/llm_args.py

```diff
@@ -2359,6 +2443,18 @@ def validate_batch_wait_timeout_ms(self) -> 'TorchLlmArgs':
             raise ValueError("batch_wait_timeout_ms must be greater than 0")
         return self
 
+    def set_mm_encoder_only(self, mm_encoder_only):
```

Comment:
This seems unnecessary, users can alter the corresponding member directly, right?